content/articles/introduction-to-gui-using-tkinter-in-python/index.md

```diff
@@ -0,0 +1,160 @@
+  
+Add the following code.
+
+```python
+sc1=StringVar('')
+t1=Label(screen,text='Automatic Password Generator',font=('Arial',25),fg='red',background ="bisque")
+t1.place(x=60,y=0)
+t2=Label(screen,text='password:',font=('Arial',14),background ="bisque")
+t2.place(x=145,y=90)
+```
+
+We have defined the `sc1` variable. In order to design the GUI window's section in this project, we'll use a label, an entry box, and a button. On the GUI window, we have created a label `t1` which is the title of the label item. We type the text on the label. We have also changed the font and background of the label. Then using the x and y coordinate values, we have assigned the location of the `t1` label.
+
+We have also created a label `t2`, it's placeholder text and its background. We have also set its coordinates.
+
+Add the code below.
+
+```Python
+il=Entry(screen,font=('Arial',14),textvariable=sc1)
+il.place(x=270,y=90)
+t3=Label(screen,text='Length: ',font=('Arial',14),background ="bisque")
+t3.place(x=145,y=120)
+t4=Label(screen,text='Strength:',font=('Arial',14),background ="bisque")
+t4.place(x=145,y=155)
+c1=Entry(screen,font=('Arial',14),width=10)
+c1.place(x=230,y=120)
+c2=Combobox(screen,font=('Arial',14),width=15)
+c2['values']=('Low Strength','Medium Strength','High Strength')
+c2.current(1)
+c2.place(x=237,y=155)
+b=Button(screen,text='Generate',font=('Arial',14),fg='red',background ="white",command=gen)
+b.place(x=230,y=195)
+
+screen.mainloop()
+```
+
+On the GUI window, we have defined the entry box with the object `il`. We have calibrate the font of `il` and put the text variable to store the string worth of `sc1`. We have used x and y coordinates to assign the location of the `il` entry. We then defined the label `t3`. Then, using coordinate values, we assigned it a location.
+
+We then created the `t4` label. Using coordinates values, we assigned the location. We also gave entry `c1` a name. Set the `Combobox` c2's values to "Low Strength", "Medium Strength", and "High Strength". The `current()` function is used to set the value of `c2`. 
+
+Then, using x and y coordinate values, we assigned the location of the `c2` `Combobox`. We define `b` as a button and use the generator command to conduct an action when the button is clicked.
+
+Then, we use x and y coordinates to assign the location of the button `b`. The `mainloop()` function is an infinite loop that is used to run the program.
+
+Here is an image of the password generator interface.
+
+![password image](/engineering-education/introduction-to-gui-using-tkinter-in-python/password.png)
+
+Congratulations on finishing this tutorial.
+
```

Comment:
@majiezra Please add a link to your code on GitHub.

Reply:
done 
@geoffrey45